isis/src/mro/apps/marcical/marcical.cpp

```diff
@@ -347,21 +337,14 @@ namespace Isis {
     PvlKeyword &colOff = icube.label()->findGroup("Instrument", Pvl::Traverse)["ColorOffset"];
     int colorOffset = toInt(colOff[0]);
 
-    for (int filter = 0; filter < numFilters; filter++) {
+    for (int i = 0; i < numFilters; i++) {
```

Comment:
filter variable is set previously, it did not seem to be resetting the larger variable, but changed for readability